script/build_all.sh

```diff
@@ -10,6 +10,6 @@ then
 fi
 
 cd $SCRIPT_PATH/../src/mtool/ui
-npm install
+npm install --legacy-peer-deps
```

Comment:
Multiple parsing were there on other places, not sure this will impact on other APIs

Reply:
Here it is trying to parse and already parsed json object. The Volumes were not listing because of this. After ng it, volumes are being listed correctly